test/e2e/fixture.go

```diff
@@ -0,0 +1,283 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func ensureAntreaRunning(data *TestData) error {
+	log.Println("Applying Antrea YAML")
+	if err := data.deployAntrea(); err != nil {
+		return err
+	}
+	log.Println("Waiting for all Antrea DaemonSet Pods")
+	if err := data.waitForAntreaDaemonSetPods(defaultTimeout); err != nil {
+		return err
+	}
+	log.Println("Checking CoreDNS deployment")
+	if err := data.checkCoreDNSPods(defaultTimeout); err != nil {
+		return err
+	}
+	return nil
+}
+
+func teardownTest(tb testing.TB, data *TestData) {
+	// exportLogs(tb, data, "beforeTeardown", true)
```

Comment:
is this line supposed to be commented out? (sorry for the pedant note, not sure if it's still commented out as the PR is WIP)

Reply:
fixed. 

Regarding your question above, we had a discussion earlier with @edwardbadboy and decided to start with having an local copy in theia. The issue with importing as lib requires antrea side to export a whole bunch of functions (framework.go, receiver of TestData), which implies 
- a revised abstraction of TestData so the current variants (antrea, multi cluster and theia) still work and make sense
- a better collected of actual tooling methods, which currently scatters all over the place in antrea (under e2e package, but defined in multiple files and certain ones relies on K8s_Utils which is not used here)

I think at this stage we still go with copying related code and trimming it down to only necessary parts, to have a functional test framework to begin with, then come back and decide whether we need to extract some common interface between antrea and theia.

Reply:
Understood... I'll make sure to follow up on this :)